test/pamela/HTN/statements.main.tpn.edn

```diff
@@ -107,16 +65,16 @@
   :tpn-type :activity,
   :uid :act-32},
  :act-9
- {:args [:field1.:simple2],
-  :argsmap {"x" :field1.:simple2},
+ {:args [field1],
```

Comment:
- For multi-level argvals, we've lost part of the reference (`simple2` in this case).  

When addressing #81, we will want to expand this (`field1.simple2`) to a structure